lib/adapters/mongodb/index.js

```diff
@@ -134,14 +98,13 @@ module.exports = function (options) {
     }
 
     const processSchema = function (hhSchema) {
-
         if (!models[hhSchema.type]) {
 
             // clean up existing models and schemas
-            delete mongoose.models[hhSchema.type]
-            delete mongoose.modelSchemas[hhSchema.type]
+            //delete db.models[hhSchema.type]
```

Comment:
removed the commented code in a new commit
the commit message describes why we no longer need this : 

removed commented obsoleted code as we no longer have risk of clashes during tests
nowadays we allocate connections with mongoose.createConnection rather than mongoose.connect
models and schemas are scoped to a specific db object and are cleared out and reinstantiated on every test run